sys/net/if.c

```diff
@@ -101,12 +101,16 @@
 
 #include <security/mac/mac_framework.h>
 
+#if !__has_feature(capabilities)
 /*
  * Consumers of struct ifreq such as tcpdump assume no pad between ifr_name
  * and ifr_ifru when it is used in SIOCGIFCONF.
+ * XXX-AM: We seem not to care about this for CheriABI so the assertion is
+ * disabled for the CHERI kernel.
  */
 _Static_assert(sizeof(((struct ifreq *)0)->ifr_name) ==
     offsetof(struct ifreq, ifr_ifru), "gap between ifr_name and ifr_ifru");
+#endif
```

Comment:
I believe we can remove this change. It's a relic of CHERI-256 (`sizeof(ifs_name) == 16`).

Reply:
I believe you are correct. Dropping it.